index.html

```diff
@@ -156,11 +156,11 @@ <h2>Introduction</h2>
       // maybe disconnect after processing the events.
       observer.disconnect();
     });
-    // retrieve buffered events and subscribe to new events
-    // for Resource-Timing and User-Timing
+    // subscribe to new events for Resource-Timing and User-Timing
+    // and retrieve buffered events from Resource-Timing 
```

Comment:
It's probably worth spelling out explicitly what's going on here. Specially, that providing names to `entryTypes` enables monitoring for said type, initialized with some default settings (e.g. buffered = false). Alternatively, you can specify the entryType name as a key and provide a dictionary with custom settings — e.g. `resource : {buffered: true}`. 

On that note, @toddreifsteck @yoavweiss would love to get your sanity check and review on this pattern.